### PR TITLE
makes kittens have randomized names

### PR DESCRIPTION
**Pull request by UselessTheremin**
half these names are a crime against god what am i doing
all jokes aside just thought this could be interesting because whenever runtime makes a bunch of kittens you can't tell em apart, feel free to suggest other names if you want

:cl:  
rscadd: kittens now have randomized names
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -80,6 +80,14 @@
 	pass_flags = PASSMOB
 	mob_size = MOB_SIZE_SMALL
 	collar_type = "kitten"
+	var/list/pet_kitten_names = list("Fajita", "Pumpkin", "Meowchael", "Catrick", "Genghis Kat", "Sir Isaac Mewton", "Nugget", "Meowchelangelo", "Meowgaret", "Lemon", "Meowria", "Todd Meoward", "Dolly Purrton", "Pickle", "Runt", "Claws")
+	var/list/rare_pet_kitten_names = list("Fuckface", "Chief Meowdical Officer", "Mewcular Opurrative")
+
+/mob/living/simple_animal/pet/cat/kitten/initialize()
```
Comment (Absolucy) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]:
```suggestion
/mob/living/simple_animal/pet/cat/kitten/Initialize()
```

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 2]
```diff
@@ -80,6 +80,14 @@
...
+	if(prob(5))
```
Comment (Absolucy) [inline code/modules/mob/living/simple_animal/friendly/cat.dm:88]:
```suggestion
	. = ..()
	if(prob(5))
```

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 3]
```diff
@@ -80,6 +80,14 @@
...
+			name = pick(rare_pet_kitten_names)
```
Comment (Absolucy) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]:
```suggestion
		name = pick(rare_pet_kitten_names)
```

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 4]
```diff
@@ -80,6 +80,14 @@
...
+		else
```
Comment (Absolucy) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]:
```suggestion
	else
```

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 5]
```diff
@@ -80,6 +80,14 @@
...
+			name = pick(pet_kitten_names)
```
Comment (Absolucy) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]:
```suggestion
		name = pick(pet_kitten_names)
```

**Review by Absolucy — CHANGES_REQUESTED** [review]

**Comment by UselessTheremin** [comment]
thank you lucy ❤️❤️❤️
sorry i cannot code lol
:^)

**Comment by Sniblet** [comment]
Patches, Quilt, Skippy, Teddy, Frank
Rare: Dumbass Cat

**Comment by UselessTheremin** [comment]
> Patches, Quilt, Skippy, Teddy, Frank Rare: Dumbass Cat

added!

**Review by adamsong — APPROVED** [review]
Will merge after 24 hours

**Review by Bibby0110 — APPROVED** [review]

**Comment by Hopekz** [comment]
Sloppy PR since we usually use .txt files for name lists but if the maintainers are okay with it I can't really complain

**Comment by UselessTheremin** [comment]
> Sloppy PR since we usually use .txt files for name lists but if the maintainers are okay with it I can't really complain

am lazy and didnt feel like making an entirely new file for that and also the names for punpun dont use a .txt file, if you want to do it be my guest lol

**Pull request merged**